CHANGELOG.md

```diff
@@ -162,6 +162,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 
 
+- Deprecated `add_to_queue`, `get_from_queue` from Lightning Module ([#9126](https://github.com/PyTorchLightning/pytorch-lightning/pull/9126))
```

Comment:
```suggestion
- Deprecated `add_to_queue`, `get_from_queue` from Lightning Module in favor of overriding the same in the DDPSpawnPlugins ([#9126](https://github.com/PyTorchLightning/pytorch-lightning/pull/9126))
```

Reply:
actually I am proposing we deprecate these functions and dont offer an alternative, see my comment in https://github.com/PyTorchLightning/pytorch-lightning/pull/9118#issuecomment-907429486

do you think that's possible?

Reply:
these methods were introduced in 1.4.4. If they are undocumented (it seems not mentioned in changelog) then maybe it's ok to remove them but generally speaking I think we cannot just remove API without offering an alternatie.